src/fondant/compiler.py

```diff
@@ -226,7 +226,7 @@ def _resolve_imports(self):
             self.kfp = kfp
         except ImportError:
             msg = """You need to install kfp to use the Kubeflow compiler,\n
-                     you can install it with `pip install --extras pipelines`"""
+                     you can install it with `pip install --extras kfp`"""
```

Comment:
I think the extras is still called `pipelines`. You can however remove the `k8s` dependency from it since we're no longer using it and maybe rename `pipelines` to `kfp`

Reply:
I will just update this and rename the `pipelines` extra to `kfp` in a different PR